src/schema/rules/sidecars/eyetrack.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+---
+EyeTrackingHardware:
+  selectors:
+    - datatype == "eyetrack"
```

Comment:
Why datatype? - this will probably go away, but in case it regresses, we should not set datatype here.
```suggestion
```